bin/build-release

```diff
@@ -143,9 +154,13 @@ tstart=${_RET}
 
 # Stage 1: DOWNLOAD
 logevent "start download" -
-rm -f download
-mkdir -p ../download
-ln -snf ../download download
+if [ "$CI_BUILD" = "true" ]; then
+    mkdir -p download
+else
+    rm -f download
+    mkdir -p ../download
+    ln -snf ../download download
```

Comment:
I've always found it a bit unexpected to create stuff outside the source tree, I assume the idea was being able to easily share the downloads when building from multiple source trees on the same host? I'd suggest to just go with a static "download/.gitkeep" added and then this whole block can be dropped.

Reply:
I think we can do it as part of 0.6 release while keeping current way for 0.5

Reply:
Sounds reasonable, o.k.

Reply:
https://github.com/cirros-dev/cirros/pull/82/commits/9bd9065ed80ee1ae58c6b25b7f348e018ff36376 does it